apps/rule-manager/client/src/ts/components/RulesTable.tsx

```diff
@@ -104,7 +126,8 @@ const RulesTable = () => {
                 }
             })
     }
-    const columns = createColumns(openEditRulePanel);
+    const pageData = useContext(PageContext);
+    const columns = createColumns(openEditRulePanel, pageData ? hasCreateEditPermissions(pageData.permissions) : false);
```

Comment:
Non-blocking: you could move the permissions hooks within `createColumns`, and save passing an argument here.

Reply:
Now done!